docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #60.        if np.any((decay &lt;= 0) | (decay &gt;= 1)):
</pre><p>Maybe instead of raising an error, we can just clip the values:</p><p><code>np.clip(decay, a_min=0.0, a_max=1.0)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
Yes. That can work. there should be no problem. I will try it and if I notice a problem, I will let you know (I just want to make sure no problem will appear when decay is set to 0 or 1.)

Reply:
<p>Okay, so I think my issue was with the readability of the line. I think I would be okay with raising the value error as you had if the condition were:</p><p> </p><pre class="ql-syntax" spellcheck="false">if np.any(decay &lt;= 0) or np.any(decay &gt;= 1):
    raise ValueError("All <code>decay</code> values must be between 0.0 and 1.0")
</pre><p>To me, this is more straightforward to parse out the meaning without thinking too much. I also changed/shortened the error message</p>